### ISSUE TITLE
Alternative download source

### ISSUE THREAD

**Issue by IzzySoft** [nominalista/expenses#4]
First thanks for providing your light-weight, tracker-free app open source! Would you consider making the APK available outside of Google Play – for those who cannot use Play Store, or avoid it for privacy reasons? You could e.g. [tag your releases](com.nominalista.expenses) – and then attach the APK to the resp. release, or have it built and listed by [F-Droid](https://f-droid.org/).

**Comment by nominalista** [comment]
Good idea. I added APK to [releases page](https://github.com/nominalista/expenses/releases). 

About F-Droid, I've recently added Firebase dependency to the project and I'm concerned about [Inclusion Policy](https://f-droid.org/en/docs/Inclusion_Policy/). What do you think?

**Comment by IzzySoft** [comment]
Thanks a lot! But why does that APK now come with Firebase Analytics? That's a show-stopper – with a finance app, even for my repo. Firebase itself is a show-stopper for the official repo, and I'd rather avoid it in a finance app as well (it's proprietary, so you cannot tell what it really does). Same goes for GMS, which of course comes along if you integrate Firebase.

I was just about to add your app to my repo as a first step (it's often a "stepping stone" on the way to the official one) – but with those dependencies, I unfortunately can't. For a compromise, would you consider a FOSS build flavor (that comes without the 3 mentioned dependencies), and attach that APK as well (so I can pick and verify; disclosure: I'm one of the F-Droid maintainers)?

**Comment by nominalista** [comment]
It was not my intention to build real FOSS application so far. My intention was to create simple tool that could become useful for other user, like it is useful for me on my daily basis. I've made it open source so other developers could learn from it. I also wanted to integrate with Firebase to make online synchronization for multiple devices. Firebase makes it much easier.

BUT I noticed some requests from community to submit app to F-Droid. If it is really useful to have fully open source and non-tracking finance app, I will be happy to do so. 

Please contact me ([the.nominalista@gmail.com](mailto://the.nominalista@gmail.com)) so we could figure out together how to make build flavor more friendly and fully free and open-source.

**Comment by IzzySoft** [comment]
Great, thanks! As I'm no Android dev, I'm not a good help with the technical details (and for privacy reasons rather avoid writing to gmail.com addresses as I don't want Google to scan my mails or even collect details on who I communicate with).

There are plenty of examples for apps providing separate build flavors for F-Droid and Play. I could give you some of them to compare, if that would help you.

**Comment by aqeeliz** [comment]
Any update on this? I (with I am sure, many others) would really like a F-Droid release.
